### ISSUE TITLE
Build fails on GNOME 41.1

### ISSUE THREAD

**Issue by DarkPhoenix645** [F-i-f/soft-brightness#44]
I am well aware that the extension does not currently support GNOME 41, but I did try to do a manual install and it seems the build process fails at the `meson install` command. Here's the output of the build:

```
ninja: Entering directory `build'
[0/1] Installing files.
Installing gschemas.compiled to /home/nilanjan/.local/share/gnome-shell/extensions/soft-brightness@fifi.org/schemas
Installing /tmp/soft-brightness/schemas/org.gnome.shell.extensions.soft-brightness.gschema.xml to /home/nilanjan/.local/share/gnome-shell/extensions/soft-brightness@fifi.org/schemas
Installing /tmp/soft-brightness/src/extension.js to /home/nilanjan/.local/share/gnome-shell/extensions/soft-brightness@fifi.org
Installing /tmp/soft-brightness/src/prefs.js to /home/nilanjan/.local/share/gnome-shell/extensions/soft-brightness@fifi.org
Installing /tmp/soft-brightness/src/utils.js to /home/nilanjan/.local/share/gnome-shell/extensions/soft-brightness@fifi.org
Installing /tmp/soft-brightness/build/metadata.json to /home/nilanjan/.local/share/gnome-shell/extensions/soft-brightness@fifi.org
Installing /tmp/soft-brightness/meson-gse/lib/convenience.js to /home/nilanjan/.local/share/gnome-shell/extensions/soft-brightness@fifi.org
Installing /tmp/soft-brightness/meson-gse/lib/logger.js to /home/nilanjan/.local/share/gnome-shell/extensions/soft-brightness@fifi.org
Installing /tmp/soft-brightness/dbus-interfaces/org.gnome.Mutter.DisplayConfig.xml to /home/nilanjan/.local/share/gnome-shell/extensions/soft-brightness@fifi.org/dbus-interfaces
Running custom install script '/tmp/soft-brightness/meson-gse/meson-scripts/install-po soft-brightness /home/nilanjan/.local/share/gnome-shell/extensions/soft-brightness@fifi.org/locale'
--- stdout ---
Unknown subcommand.

--- stderr ---

FAILED: install script '/tmp/soft-brightness/meson-gse/meson-scripts/install-po soft-brightness /home/nilanjan/.local/share/gnome-shell/extensions/soft-brightness@fifi.org/locale' exit code 1, stopped
FAILED: meson-install 
/usr/bin/meson install --no-rebuild
ninja: build stopped: subcommand failed.

```

PS: appreciate the awesome work that you do and I love this extension. It is also only way I have to control my laptop's screen brightness, so please add 41 support as soon as possible.

**Comment by DarkPhoenix645** [comment]
For some reason, 10 minutes after I made this issue, the extension showed up in the Extensions app and is currently working properly.

**Comment by F-i-f** [comment]
The build issue has been addressed as of [8013e60](https://github.com/F-i-f/soft-brightness/commit/8013e60b728e2b01fd36847192896b3afdaa1d7f).
This was a separate issue.
